nicegui/json/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Custom json module. Provides dumps and loads implementations
+wrapping orjson.
+
+Custom module required in order to override json-module used
+in socketio.AsyncServer.
+"""
+
+from typing import Any
+
+import orjson
+from fastapi import Response
+
+ORJSON_OPTS = orjson.OPT_SERIALIZE_NUMPY | orjson.OPT_NON_STR_KEYS
+
+
+def dumps(obj, sort_keys=False, separators=None):
```

Comment:
Why introducing `sort_keys` and `separators` that are not used?

Reply:
I have added more comments to the code to clarify it. It matches Python's `json.dumps` API, and some internals of `AsyncServer` call it like this, so we need to mimic the function. I have added support for `sort_keys`, though probably never used anywhere.